<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyber druh pizzy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vyber nápoj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cena">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Pizza</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Navíc</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>103</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Sýr 10 Kč</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>Olivy 15 Kč</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>Feferonky 15 Kč</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>105</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>šunková 75 Kč</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>salámová 85 Kč</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>Havaii 110 Kč</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>špenátová 95 Kč</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Nápoje</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <widget class="QCheckBox" name="checkBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Džus 0,2l 15 Kč</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pivo 0,5 l 30 Kč</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Coca-Cola 0,5 l 35 Kč</string>
    </property>
   </widget>
   <zorder>checkBox_6</zorder>
   <zorder>checkBox_7</zorder>
   <zorder>checkBox_8</zorder>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_cena</zorder>
  <zorder>checkBox</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>radioButton</zorder>
  <zorder>radioButton_2</zorder>
  <zorder>radioButton_3</zorder>
  <zorder>radioButton_4</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>469</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_6</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_7</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_8</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_4</sender>
   <signal>toggled(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>recap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>18</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>recap()</slot>
 </slots>
</ui>
